doc/sphinx/source/quickstart/installation.rst

```diff
@@ -392,3 +428,159 @@ To do this, run the tool with:
 
 If everything was installed properly, ESMValTool should have printed a
 help message to the console.
+
+Using the development version of the ESMValCore package
+-------------------------------------------------------
+
+If you need the latest developments of the ESMValCore package, you
+can install it from source into the same conda environment.
+
+.. attention::
+    The recipes and diagnostics in the ESMValTool repository are compatible
+    with the latest released version of the ESMValCore.
+    Using the development version of the ESMValCore package is only recommended
+    if you are planning to develop new features for the ESMValCore, e.g.
+    you want to implement a new preprocessor function.
+
+First follow all steps above.
+Next, go to the place where you would like to keep the source code and clone the
+ESMValCore github repository:
+
+.. code-block:: bash
+
+    git clone https://github.com/ESMValGroup/ESMValCore
+
+or
+
+.. code-block:: bash
+
+    git clone git@github.com:ESMValGroup/ESMValCore
+
+The command above will create a folder called ``ESMValCore``
+containing the source code of the tool in the current working directory.
+
+Go into the folder you just downloaded
+
+.. code-block:: bash
+
+    cd ESMValCore
+
+and then install ESMValCore in development mode
+
+.. code-block:: bash
+
```

Comment:
Perhaps add a line on what the "-e" is for. Or, perhaps use "--editable" for a bit more of a hint.

Reply:
Done